### ISSUE TITLE
it's not obvious how to log out of the app

### ISSUE THREAD

**Issue by jomapp** [tutao/tutanota#10668]
### What happened?

We should use a different label for the logout button on the webApp.
For mobile and desktop it is currently saying "switch account" which is accurate, but for the webApp it should rather say "logout"

<img width="640" height="360" alt="Image" src="https://github.com/user-attachments/assets/196225d3-71a5-4262-9fcb-6127b1152ad3" />

**Comment by ganthern** [comment]
@jomapp and I talked about this and looking for a "Logout" button is the default for many people, so the use case is valid.

That button is a proper Logout button when you don't have stored credentials, but does in fact just allow to switch accounts if you do. The actual logout button is the "Delete" text that appears in the credentials selector after selecting "remove account". This is independent from the platform and should be improved.
